Feature/remove direct emdk dependency

esotrifork opened the pull request:
To avoid having our project depend on emdk as well as this lib, we create a wrapper class for the errors returned in some cases

esotrifork committed:
Avoid using emdk specific classes for return values as that would introduce a dependency to the emdk library for apps using this lib

esotrifork committed:
bump version

jhetrifork commented on app/build.gradle:
```diff
@@ -65,14 +65,10 @@ dependencies {
 
-//    def emdk_version = "1.0.0"
+//    def emdk_version = "1.0.1"
 //    implementation "com.github.gls-denmark:kotlin-emdk:$emdk_version"
```
If you have this in the documentation, I think you can delete it from here, so you do not have to update it each time.

esotrifork replied:
I have linked to jitpack so users can see what the latest version is and use that instead of us having to bump them each time

esotrifork committed:
Added guide to error during startup

esotrifork commented on kemdk/build.gradle:
```diff
@@ -11,7 +11,7 @@ plugins {
 }
 
-version = "0.0.1"
+version = "1.0.1"
```
Releases are based on git tag. This might have no effect at all. Do we need to keep it for reasons?

esotrifork replied:
Since it is not used, we remove it from code to avoid having a version number that we need to bump for documentation purpose only

esotrifork committed:
Avoid having unnecessary references to versions that we have to remember to bump, but have no effect

To avoid having to bump versions in comments and build.gradle (it is not used for release versions anyway) we remove them all together and link to jitpack instead for getting latest version
